content/docs/guides/benchmarking-latency.md

```diff
@@ -0,0 +1,107 @@
+
+## Understanding cold vs. hot queries
+
+When benchmarking Neon databases, you'll encounter two distinct types of queries:
+
+- **Cold queries**: Occur when a previously suspended compute resource is activated to process a request. By default, free-tier Neon databases suspend after 5 minutes of inactivity. A cold query typically adds several hundred milliseconds of latency.
+- **Hot queries**: Execute on already-active database instances, providing a more consistent performance baseline with normal low latency.
+
+On paid plans, you can configure or disable the auto-suspend timeout to customize your testing approach or eliminate cold starts entirely. See [Compute Lifecycle](/docs/introduction/compute-lifecycle) and [Auto-suspend Configuration](/docs/introduction/auto-suspend) for more details.
+
+## Benchmarking Methodology
+
+For accurate benchmarking, always measure cold and hot queries separately:
+
+1. **Cold query testing**:
+
+   - Ensure your database is in a suspended state (idle for at least five minutes on free tier)
+   - Make a request to trigger compute activation
+   - Measure this connection time, which will include the startup overhead
+
+2. **Hot query testing**:
+   - After triggering compute activation with a cold query
+   - Make subsequent requests within the five-minute window
+   - Measure these connection times, which reflect normal operation
+
+This methodology isolates the cold start overhead from normal operating performance, giving you a clearer picture of both typical performance versus worst-case latency.
+
+## Testing environment considerations
+
+Before running benchmarks, determine exactly what kind of latency you want to measure:
+
+- **Server-to-database latency**: If you're testing how quickly your application server can communicate with Neon, make your requests from the same location as your server. This is typically the most relevant metric for API performance.
+
+- **Client-to-database latency**: If you're testing direct client connections (rare in production), benchmark from client locations.
+
+Once you've determined what you're measuring:
+
+- **Match your test environment to production**: Run your tests from the same region (or as close as possible) to your production environment. If your Neon database is in `us-east-1`, execute your benchmarks from a server in the same AWS region.
+
+- **Avoid localhost testing**: Testing from your local workstation doesn't reflect real-world conditions where the database is typically queried from your deployed server, not the client.
+
+- **Geographic proximity**: Client-database proximity is the primary factor in connection latency. Testing from the same region will yield the most accurate results.
```

Comment:
Is this the same point as the first one about running tests in the same region?

Reply:
Good point, this will be addressed 👍